### PR TITLE
Add inline source maps to compiled lib

### PR DESCRIPTION
**Pull request by mgcrea**
##### Checklist

- [X] tests and code linting passes
- [X] a test is included
- [X] documentation is changed or added
- [X] the commit message follows commit guidelines

##### Affected core subsystem(s)

Babel Compilation

##### Description of change

Added inline sourcemaps to the lib built files to get proper stack traces:

from:
```
/Users/olivier/Developer/nodejs/ied/ied/lib/registry.js:97
  _assert2.default.equal(statusCode, 200, 'error status code ' + uri + ': ' + error);
                   ^
AssertionError: error status code https://registry.npmjs.org/eslint/node_modules/eslint: undefined
    at checkStatus (/Users/olivier/Developer/nodejs/ied/ied/lib/registry.js:97:20)
    at SafeSubscriber._next (/Users/olivier/Developer/nodejs/ied/ied/lib/registry.js:134:12)
    at SafeSubscriber.__tryOrSetError (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:247:10)
    at SafeSubscriber.next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:191:23)
    at Subscriber._next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:135:22)
    at Subscriber.next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:95:12)
    at DoSubscriber._next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/operator/do.ts:93:20)
```

to:
```
../src/registry.js:48
	assert.equal(statusCode, 200, `error status code ${uri}: ${error}`)
        ^
AssertionError: error status code https://registry.npmjs.org/supertest/node_modules/supertest: undefined
    at checkStatus (../src/registry.js:48:9)
    at SafeSubscriber._next (../src/registry.js:80:19)
    at SafeSubscriber.__tryOrSetError (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:247:10)
    at SafeSubscriber.next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:191:23)
    at Subscriber._next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:135:22)
    at Subscriber.next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:95:12)
    at DoSubscriber._next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/operator/do.ts:93:20)
    at DoSubscriber.Subscriber.next (/Users/olivier/Developer/nodejs/ied/ied/node_modules/rxjs/src/Subscriber.ts:95:12)
    at ../src/util.js
```

### CODE REVIEW

File: src/cmd.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import minimist from 'minimist'
 import * as config from './config'
+import 'source-map-support/register'
```
Comment (alexanderGugel) [inline src/cmd.js]: Does this have performance implications?

File: Makefile [hunk 2]
```diff
@@ -56,4 +56,4 @@ dev:
 	$(DEPS_BIN_DIR)/mocha test/*-test.js $(MOCHA_FLAGS) --watch
 
 watch:
-	$(DEPS_BIN_DIR)/babel -w src --out-dir lib
+	$(DEPS_BIN_DIR)/babel -w src --out-dir lib -s inline
```
Comment (alexanderGugel) [inline Makefile]: I definitely see the value in source maps here, but I think we should only have it in the `watch` target, which is used during development.

Comment (alexanderGugel) [inline Makefile]: Can you prefix the script with `NODE_ENV=dev` or something similar and then only `require` in `'source-map-support/register'` if `NODE_ENV === 'dev'`.

**Comment by mgcrea** [comment]
@alexanderGugel should be good now. Tell me if you see anything else.

**Comment by alexanderGugel** [comment]
Thanks.

**Pull request merged**